bundles/core/src/main/java/com/adobe/cq/email/core/components/util/StyleTokenFactory.java

```diff
@@ -133,11 +134,13 @@ public static void addProperties(StyleToken styleToken, String properties) {
      * @param styleToken the {@link StyleToken}
      * @return a valid CSS representation
      */
+    @Nullable
     public static String toCss(StyleToken styleToken) {
-        if (Objects.isNull(styleToken)) {
-            return null;
+        String css = null;
+        if (Objects.nonNull(styleToken)) {
+            css = styleToken.getSelector() + " { " + getAllProperties(styleToken) + " }\n";
```

Comment:
You can probably replace the "\n" character at line 141 with another space

Reply:
I will keep the new line for now for each selector like this is done by the MJML framework. If we see issues with the JSON exchange between AEM and ACC we should probably remove it.